resources/lang/en-US/general.php

```diff
@@ -158,7 +158,7 @@
     'image_upload'         	=> 'Upload Image',
     'filetypes_accepted_help'    => 'Accepted filetype is :types. Max upload size allowed is :size.|Accepted filetypes are :types. Max upload size allowed is :size.',
     'filetypes_size_help'   => 'Max upload size allowed is :size.',
-    'image_filetypes_help'  => 'Accepted filetypes are jpg, webp, png, gif, svg, and avif. Max upload size allowed is :size.',
+    'image_filetypes_help'  => 'Accepted filetypes are :filetypes. Max upload size allowed is :size.',
```

Comment:
I think we already have this string as `filetypes_accepted_help` in the same file, but we pass it `:types: instead of `:filetypes`. We can probably merge these?

Reply:
done 👍